bot/cogs/eggs/reaction.py

```diff
@@ -0,0 +1,27 @@
+from discord.ext.commands import Cog
+from datetime import datetime, timedelta
+
+
+class Reaction(Cog, name="Eggs"):
+    def __init__(self, client):
+        self.bot = client
+        self.THRESHOLD = timedelta(seconds=1.5)
```

Comment:
I suggest turning this into a [class variable](https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables), because
1) it's global to the class itself, it doesn't need to be bound to an instance of an object of the type, and
2) provides better readability.

Like
```py
class Reaction(Cog, name="Eggs"):
	THRESHOLD = timedelta(seconds=1.5)

	def __init__(self, client):
		# ...
```

Reply:
> it's global to the class itself, it doesn't need to be bound to an instance of an object of the type

Everything in this class can be global.